plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -1830,12 +1831,22 @@ def validate_radio_parameters(self, radio_config, radio_series, errormsg):
                     )
 
         channel_width = radio_config.get("channel_width")
-        if channel_width and channel_width not in ("20 MHz", "40 MHz", "80 MHz", "160 MHz", "320 MHz"):
-            errormsg.append(
-                "channel_width: Invalid value '{0}' for Channel width in playbook. "
-                "Must be one of: '20 MHz', '40 MHz', '80 MHz', '160 MHz', or '320 MHz'."
-                .format(channel_width)
-            )
+        if channel_width and channel_width in ("20 MHz", "40 MHz", "80 MHz", "160 MHz", "320 MHz") and\
```

Comment:
Can you simplify the logic in your code for checking the channel_width and its applicability based on the radio_series. This refactoring reduces redundancy and improves readability. 

```
valid_channel_widths = ("20 MHz", "40 MHz", "80 MHz", "160 MHz", "320 MHz")

if channel_width:
    if radio_series == "2.4ghz_radio":
        errormsg.append("channel_width is not applicable for the 2.4GHz radio")
    elif channel_width not in valid_channel_widths:
        if radio_series == "6ghz_radio":
            errormsg.append(
                "channel_width: Invalid value '{0}' for Channel width in playbook. "
                "Must be one of: {1}.".format(channel_width, ", ".join(valid_channel_widths)))
        else:
            errormsg.append(
                "channel_width: Invalid value '{0}' for Channel width in playbook. "
                "Must be one of: '20 MHz', '40 MHz', '80 MHz' or '160 MHz'.".format(channel_width))

```

Reply:
modified also added additional validation